ghost/core/core/server/api/endpoints/actions.js

```diff
@@ -4,6 +4,9 @@ module.exports = {
     docName: 'actions',
 
     browse: {
+        headers: {
```

Comment:
None of the GET methods should ever reset the cache. I think it mostly makes sense to apply the header to the write methods like POST/PUT/DELETE methods.